### ISSUE TITLE
model training

### ISSUE THREAD

**Issue by AidenFather** [Syazvinski/Traffic-Light-Detection-Color-Classification#1]
Hi, 

Can you share what dataset did you use for the model training?

Thanks,

**Comment by Syazvinski** [comment]
It was a mix of dozens of smaller datasets I combined. Ill try and find the original sources and get back to you here.

**Comment by AidenFather** [comment]
> It was a mix of dozens of smaller datasets I combined. Ill try and find the original sources and get back to you here.

I have tested your trained weight. It seems you included bosch and LISA datasets (correct me if am wrong). Just wonder what other datasets are used in the training. It would be appreciated if you could check and share the list of datasets. Thank you in advance!

**Comment by Syazvinski** [comment]
Yes, I do remember including those. This project was from a while back. If I have the time, I will try and compile the list, but I can confirm I did use both of those.
